website/docs/r/spring_cloud_service.html.markdown

```diff
@@ -75,6 +77,20 @@ The following arguments are supported:
 
 The `config_server_git_setting` block supports the following:
 
+* `service_runtime_subnet_id` - (Required) Specifies the subnet id to host Azure Spring Cloud Service Runtime.
+
+* `app_subnet_id` - (Required) Specifies the subnet id to host Azure Spring Cloud Apps.
```

Comment:
what does this mean in practice?

Reply:
"(Required) Specifies the full qualified resource id of an Azure Virtual Network Subnet, which will be used to host the Spring Boot microservices Applications of the Azure Spring Cloud Service Instance to be created."
How do you think?

Reply:
(as above) we avoid the phrase Resource ID since it's ambiguous - but this could be:

> (Required) Specifies the ID of the Subnet which should host the Spring Boot Applications deployed in this Spring Cloud Service. Changing this forces a new resource to be created.

Reply:
done